deployment/helm/node-feature-discovery/templates/topologyupdater.yaml

```diff
@@ -65,20 +76,29 @@ spec:
       - name: host-sys
         hostPath:
           path: "/sys"
+          type: Directory
```

Comment:
nit: unrelated. For completeness/consistency it would make sense to add the type field to all `hostPath` mounts in all templates. Maybe you could submit a separate PR about this?

Reply:
Agree!